Code/Source/Camera/CameraSensor.cpp

```diff
@@ -49,7 +50,18 @@ namespace ROS2
         AZ_Assert(!cameraName.empty(), "Camera name cannot be empty");
     }
 
+    std::array<double, 9> CameraSensorDescription::MakeCameraIntrinsics() const
+    {
```

Comment:
A comment on what is happening here would be useful

Reply:
Added comment :+1: Is this level of detail ok?

Reply:
Perfect, thank you